Makefile.prow

```diff
@@ -15,6 +38,44 @@ unit-tests:
 	fi
 	npm run test
 
+.PHONY: test-e2e
+test-e2e:
+	@./build/run-prow-e2e.sh
+
+.PHONY: pull-test-image
+pull-test-image:
+	make component/pull COMPONENT_NAME=$(COMPONENT_NAME)-tests
 
-e2e-tests:
-	@./build/run-prow-e2e.sh
\ No newline at end of file
+.PHONY: run-test-image-pr
+run-test-image-pr: # Suppress output as this contains sensitive information
+	@docker run \
```

Comment:
Do we need to change docker run to npx cypress run?

Reply:
No, let's try to run the test image directly.